pom.xml

```diff
@@ -15,8 +15,6 @@
     <description>Pup Buddy</description>
     <properties>
         <java.version>11</java.version>
-        <maven.compiler.source>17</maven.compiler.source>
-        <maven.compiler.target>17</maven.compiler.target>
```

Comment:
This was something I did to get it to work with my JDK. I reverted it after accidently pushing it.